www/src/components/TwitterWall/script.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs';
+import fetch from 'node-fetch';
+
+/**
+ * Script to generate a JSON file of tweets from the Twitter API.
+ * Fetches the tweeets from this timeline: https://twitter.com/alexdotjs/timelines/1441435105910796291
+ *
+ * Script has shitty typesafety but hey, it works :)
+ */
+
+const {
+  OAUTH_CONSUMER_KEY,
+  OAUTH_NONCE,
+  OAUTH_SIGNATURE,
+  OAUTH_TIMESTAMP,
+  OAUTH_TOKEN,
+} = process.env;
+
+const url =
+  'https://api.twitter.com/1.1/collections/entries.json?id=custom-1441435105910796291';
+const options = {
+  headers: {
+    accept: '*/*',
+    Authorization: `OAuth oauth_consumer_key="${OAUTH_CONSUMER_KEY}", oauth_nonce="${OAUTH_NONCE}", oauth_signature="${OAUTH_SIGNATURE}", oauth_signature_method="HMAC-SHA1", oauth_timestamp="${OAUTH_TIMESTAMP}", oauth_token="${OAUTH_TOKEN}", oauth_version="1.0"`,
+  },
+};
+
+/** Formats a date to mmm DD e.g. Jul 27 */
+const dateFormatter = new Intl.DateTimeFormat('en-US', {
+  month: 'short',
+  day: '2-digit',
+});
+
+const flattenTweets = (tweets: any, users: any) => {
+  return Object.entries(tweets).map(([tweetId, tweetData]: [any, any]) => {
+    const date = new Date(tweetData.created_at);
+    const userId = tweetData.user.id_str;
+
+    const content = tweetData.text;
+
+    // this removes the trailing line at the end of the tweet's content
+    const contentWithoutTrailingLink = content.replace(
+      /\b(http(s|):\/\/.+)\W*$/,
+      '',
+    );
+
```

Comment:
```suggestion
const escapedContent = content
    .replace(/&amp;/, '&')
    .replace(/&lt;/, '<')
    .replace(/&gt;/, '>')
    .replace(/&quot;/, '\"')
    .replace(/&#39;/, '\'');      

```

@ixahmedxi too hacky?